### ISSUE TITLE
Varya: Integrate spacing support in Layout Grid block

### ISSUE THREAD

**Issue by allancole** [Automattic/themes-workspace#24]
So I’ve been testing the Layout Grid block against Varya, and it seems to work pretty well out of the box. I don’t think we really need to add extra support, however the spacing rules in the plugin are different than the ones in Varya. For instance: 

Layout Grid:
```
.wp-block-jetpack-layout-grid.wp-block-jetpack-layout-gutter__medium {
    grid-gap: 16px;
}
```

If we were to do integrate that style into Varya, it would look like this:
```
.wp-block-jetpack-layout-grid.wp-block-jetpack-layout-gutter__medium {
    grid-gap: var(--global--spacing-unit);
}
```

Could be overkill, but it would allow the spacing system in Varya to influence the paddings and gutters in the Layout Block. I’m anticipating us using this block pretty regularly so having the spacing values synced here means they’ll align with other spacing rules that are already in use in the system.

If we do this we’d also need to add a few more utility classes and CSS-variables to support the spacing options that come with the plugin. 

```
--global--spacing-none: 0px;
--global--spacing-small: ??px;
--global--spacing-medium: 16px; // same as --global--spacing-unit
--global--spacing-large: ??px;
--global--spacing-huge: ??px;
```

Curious, if you all think this is worth the effort (which wouldn’t be a lot of effort) @jffng @kjellr

**Comment by jffng** [comment]
The layout grid seems worth supporting to me. Would this just mean adding another directory in `assets/sass/block/layout-grid` along with a couple sass files?

**Comment by allancole** [comment]
> Would this just mean adding another directory in assets/sass/block/layout-grid along with a couple sass files?

Precisely. To do so, you’d just need to go through [the block’s CSS](https://plugins.trac.wordpress.org/browser/layout-grid/trunk/blocks/front.css), and pick out the padding/margin/gutter rules replace them with the appropriate Varya CSS-variables—but only when it makes sense. Feel free to take a stab at in a PR :-)

**Comment by jffng** [comment]
Solved by #29.
